### DISCUSSION TITLE
Shortcut to Toggle the Sidebar Width (Toolbar -> Expanded)

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#792] [Ideas]
### What feature would you like to see?

We need a shortcut to toggle the sidebar's width.

I suggest picking a default which is similar to the Toggle Sidebar shortcut currently implemented.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/578_

**Comment by JBurdik** [comment]
+1

**Comment by momintlh** [comment]
No update on this? :(

**Reply by umarhamzy** [reply]
Try `ALT + B`

**Comment by Abdullah16M** [comment]
+1

**Reply by umarhamzy** [reply]
Try `ALT + B`

**Reply by Abdullah16M** [reply]
It doesn't toggle the full width of the sidebar, `ALT + B` toggles showing/hiding the sidebar in compact mode

**Comment by phillipleroy** [comment]
+1 to that, my gosh I really need this feature.

**Comment by momintlh** [comment]
It works with `ALT+CTRL+S` if you have compact mode enabled which is `ALT+CTRL+C`

**Comment by hexvictor** [comment]
+1
I wish we could have a collapsed sidebar (only icons) and when hovered it expands to the normal size with text.
I'm using `ALT+CTRL+C` for now, but not the same.

**Reply by diegov00** [reply]
I just found this awesome [mod](https://github.com/StormAnon/zen-sidebar-expand-on-hover/tree/main) that does exactly what you want.
